sdk-lib/src/main/java/cash/z/ecc/android/sdk/exception/Exceptions.kt

```diff
@@ -123,9 +123,9 @@ sealed class BirthdayException(message: String, cause: Throwable? = null) : SdkE
     )
     class ExactBirthdayNotFoundException(height: Int, nearestMatch: Int? = null) : BirthdayException(
         "Unable to find birthday that exactly matches $height.${
-        if (nearestMatch != null)
+        if (nearestMatch != null) {
             " An exact match was request but the nearest match found was $nearestMatch."
-        else ""
+        } else ""
```

Comment:
If I understand it right, this change was made by ktlintFormat task. Is there a way to let ktlint wraps also else branch body with brackets too?

Reply:
I've found that occasionally ktlintFormat will miss something like this.  I just manually added the braces, so it should be good going forward.  (ktlint won't remove them).

This particular situation is weird, because the if statement is within a string format.  So I don't know if that contributes to some weird behavior here too.

Reply:
Yep, that can the the problem.